### ISSUE TITLE
Insights into estrogen impact in oral health & microbiome in COVID-19

### ISSUE THREAD

**Issue by SvetlanaUP** [waldronlab/BugSigDBcuration#315]
Insights into estrogen impact in oral health & microbiome in COVID-19 – Rubén Alberto Bayardo-González – BMC Microbiology
https://bmcmicrobiol.biomedcentral.com/articles/10.1186/s12866-023-03149-5

**Comment by Ademolawe** [comment]
Hello, can I be assigned this for my fourth curation?

Eniola Adebayo

**Comment by Ademolawe** [comment]
Hello,
This curation is ready for review.
https://bugsigdb.org/Study_1028
https://bugsigdb.org/Talk:Study_1028

Eniola Adebayo
Thank you.

**Comment by Scholarpat** [comment]
Hi @SvetlanaUP,

Here are the changes I've made to this curation:

- Changed the condition from 'COVID-19' to '**_microbiome measurement_**' (oral microbiome measurement is a more accurate term but is not available on BugSigDB) in experiment 2, and to _**'viral load'**_ in experiment 3.

- Only the result for the Chao1 index was reported for differential abundance microbiome. I have excluded Shannon, which was initially curated.

- Increased taxa were initially curated as decreased and vice versa in experiments 1 and 2.

- Changed signature source 'figure f' to 'figure **3**f' and 'figure i' to 'figure **3**i' for experiments 2 and 3.

The curation is complete.
Happy Friday!
